src/interviewer.t

```diff
@@ -217,17 +235,70 @@ warner_interviewer : Person 'interviewer' 'Harried Interviewer' @interview_room
 ;
 
 ++ SpecialTopic 'anime club' ['anime', 'anime club', 'japanese', 'japanese club', 'weeaboo', 'weeb']
-    ""
+    ""//"<q>私はアニメが大好きです~~~!</q>"
```

Comment:
I'll try getting around this when I have the patience to or when I think of an alternative. If anyone knows how to get this to not be a replacement block, please feel free.